portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path

```diff
@@ -147,6 +147,51 @@
 	<td>//div[label[contains(.,'Sorting')]]/select</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ANY_ELLIPSIS_BUTTON</td>
+	<td>//button[contains(@class, 'dropdown-toggle')]//*[contains(@class, 'lexicon-icon-ellipsis')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ENTRY_ELLIPSIS_BUTTON_BY_POSITION</td>
+	<td>//tr[contains(@class,'orderable-table-row')][${key_entryPosition}]//*[contains(@class, 'lexicon-icon-ellipsis')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASCENDING_SORTING_OPTION</td>
+	<td>//*[contains(@class, 'modal-dialog')]//option[contains(@value, 'asc')]</td>
+	<td></td>
```

Comment:
The other dropdown paths can't be used in this case because the structure of data set is different from the other components, so their paths don't fit this element.